ui/js/controllers/perf/alerts.js

```diff
@@ -217,7 +208,8 @@ perf.controller('AlertsCtrl', [
         };
 
         // can filter by alert statuses or just show everything
-        $scope.statuses = _.map(phAlertSummaryStatusMap);
+        $scope.statuses = Object.keys(phAlertSummaryStatusMap).map(
```

Comment:
Let's use ``Object.values`` instead.  Then I think it's a straight one-line change.